recipes/tutorials/plugins_development.ipynb

```diff
@@ -693,7 +691,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.7.15"
+   "version": "3.7.11"
```

Comment:
Why downgrade the version ?

Reply:
It's because I executed the notebook with my base python environment which is in version 3.7.11. It's metadata from a notebook, it's not displayed at all